### PR TITLE
fix: apply series for new data download

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

After the data is changed, the reflected data should be downloaded when downloading.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by jungeun-cho — APPROVED** [review]
확인했습니다~!

**Review by js87zz — APPROVED** [review]

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/chart/src/store/seriesData.ts [hunk 1]
```diff
@@ -201,7 +200,7 @@ const seriesData: StoreModule = {
         };
       });
 
-      extend(state.series, newSeriesData);
```
[the remark this answers is not in the document]
Comment (jung-han) [reply apps/chart/src/store/seriesData.ts:204]: 아 이부분은 원래대로 돌려놓겠습니다~

**Pull request merged**